application/server/lsp/message_types.go

```diff
@@ -783,3 +785,18 @@ type IacIssueData struct {
 	Path          []string `json:"path"`
 	References    []string `json:"references,omitempty"`
 }
+
+type IacIssuePathToken struct {
+	Value any
+}
+
+func (c *IacIssuePathToken) String() string {
+	switch val := c.Value.(type) {
+	case int:
+		return strconv.Itoa(val)
+	case string:
+		return val
+	default:
+		panic("invalid iac issue path component type")
```

Comment:
Please not panic and rely on logging or using `error_reporting.ErrorReporter` to [report errors](https://github.com/snyk/snyk-ls/blob/main/CONTRIBUTING.md#error-reporting).

Reply:
Fixed in https://github.com/snyk/snyk-ls/pull/269/commits/94151c1583e7f070a3985acd7224de1e3dba7ec2